java/spark-bundle/build.gradle.kts

```diff
@@ -18,7 +19,7 @@ val sparkVersion = project.properties.getOrDefault("sparkVersion", "3.1.1") as S
 val artifactBaseName = "whylogs-spark_${sparkVersion}-scala_$scalaVersion"
```

Comment:
I think we also want to update the artifact name here to be consistent with previously published bundled jar

`val artifactBaseName = "whylogs-spark-bundle_${sparkVersion}-scala_$scalaVersion"`

Reply:
We can do in a follow up since its not in your change, but up to you.